src/components/LexiconClassViewer.tsx

```
@@ -507,569 +552,780 @@ const LexiconClassViewer: React.FC<LexiconClassViewerProps> = ({
+    >
+      <div className="method-list-item-label">
+        <div className="method-list-item-header">
+          <div className="method-list-item-label-name">
+            {renderHighlightedText(getHighlightedClassName(cls))}
+          </div>
+          <button
+            className="expand-button"
+            onClick={() => toggleExpanded(index)}
+            aria-label={expandedClasses.has(index) ? 'Collapse' : 'Expand'}
+          >
+            {expandedClasses.has(index) ? '−' : '+'}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)